openstack/compute/v2/extensions/aggregates/testing/fixtures.go

```diff
@@ -60,39 +60,66 @@ const AggregateCreateBody = `
 }
 `
 
-// First aggregate from the AggregateListBody
-var FirstFakeAggregate = aggregates.Aggregate{
-	AvailabilityZone: "",
-	Hosts:            []string{},
-	ID:               1,
-	Metadata:         map[string]string{},
-	Name:             "test-aggregate1",
-	CreatedAt:        time.Date(2017, 12, 22, 10, 12, 6, 0, time.UTC),
-	UpdatedAt:        time.Time{},
+const AggregateGetBody = `
+{
+    "aggregate": {
+            "name": "test-aggregate2",
+            "availability_zone": "test-az",
+            "deleted": false,
```

Comment:
in fact you're including the fields in your output.

Reply:
This fields are used at #691 as well.